core/service.go

```diff
@@ -164,6 +176,11 @@ func (s *Service) StorageRoot() (common.Hash, error) {
 	return s.storageState.StorageRoot()
 }
 
+func (s *Service) retrieveAuthorityData() ([]*babe.AuthorityData, error) {
+	// TODO: when we update to a new runtime, will need to pass in the latest block number
```

Comment:
Looks like we should link these to an issue and update that runtime.

Reply:
will add to comment

Reply:
https://github.com/ChainSafe/gossamer/issues/563 should cover everything